main.go

```diff
@@ -120,48 +95,47 @@ func run() (int, error) {
 		output = os.Stdout
 	}
 
-	if flags.Has("q", args) {
-		q, err := flags.Value("q", args)
-		if !validArg(err, q) {
-			return 1, errors.New("invalid entry for -q \n" + usage)
-		}
-
+	if *qFlag != "" {
 		qu = TextQualifier{
 			On:        true,
-			Qualifier: q,
+			Qualifier: *qFlag,
 		}
 	}
 
-	sw := NewAligner(input, output, sep, qu)
-
-	if flags.Has("a", args) {
-		val, err := flags.Value("a", args)
-		if !validArg(err, val) {
-			return 1, errors.New("invalid entry for -a \n" + usage)
-		}
-		switch val {
-		case "left":
-			sw.UpdatePadding(PaddingOpts{Justification: JustifyLeft})
-		case "right":
-			sw.UpdatePadding(PaddingOpts{Justification: JustifyRight})
-		case "center":
-			sw.UpdatePadding(PaddingOpts{Justification: JustifyCenter})
-		default:
-			sw.UpdatePadding(PaddingOpts{Justification: JustifyLeft})
-		}
+	sw := NewAligner(input, output, *sFlag, qu)
+
+	switch *aFlag {
+	case "left":
+		sw.UpdatePadding(PaddingOpts{Justification: JustifyLeft})
+	case "right":
+		sw.UpdatePadding(PaddingOpts{Justification: JustifyRight})
+	case "center":
+		sw.UpdatePadding(PaddingOpts{Justification: JustifyCenter})
+	default:
+		sw.UpdatePadding(PaddingOpts{Justification: JustifyLeft})
 	}
 
 	lines := sw.ColumnCounts()
 
-	sw.OutputSep(outSep)
+	sw.OutputSep(*dFlag)
 	sw.Export(lines)
 
 	return 0, nil
 }
 
-func validArg(err error, arg string) bool {
-	if err != nil || arg == "" {
-		return false
+func parseFlags() {
```

Comment:
Cool you split this out however, just food for thought, the compiler is likely going to inline this call.